lib/ansible/modules/packaging/os/pkg5.py

```diff
@@ -74,6 +85,7 @@ def main():
             state=dict(type='str', default='present', choices=['absent', 'installed', 'latest', 'present', 'removed', 'uninstalled']),
             accept_licenses=dict(type='bool', default=False, aliases=['accept', 'accept_licences']),
             be_name=dict(type='str'),
+            refresh=dict(type='bool', default=True),
```

Comment:
Should there be a comma at the end? Does Python ignore this?

Reply:
@danowar2k: https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas